dynamiqs/_utils.py

```diff
@@ -41,3 +42,27 @@ def bexpect(O: Array, x: Array) -> Array:
     if isket(x):
         return jnp.einsum('ij,bjk,kl->b', dag(x), O, x)  # <x|O|x>
     return jnp.einsum('bij,ji->b', O, x)  # tr(Ox)
+
+
+def compute_batching(f, cartesian_batching, other_in_axes, out_axes, *should_batch):
```

Comment:
```suggestion
def compute_in_axes(batch_axes, cartesian_batching):
```
(and corresponding function modifications so as to return the `in_axes` argument)